src/components/settings/groups/GroupsPanel.tsx

```diff
@@ -0,0 +1,46 @@
+import { Box, Button, Divider, Flex, HStack, SimpleGrid, Spacer, Text, useDisclosure } from '@chakra-ui/react';
+import useSubjects from '../../../hooks/useSubjects';
+import { SubjectType } from '../../../entities/Subject';
+import { AddIcon } from '@chakra-ui/icons';
+
+export const GroupsPanel = () => {
+  const { isOpen: isCreateModalOpen, onOpen: onCreateModalOpen, onClose: onCreateModalClose } = useDisclosure();
```

Comment:
isOpen and onClose are not used. Also - perhaps it's better to name onOpen: setOpen, so that we know what it does instead when it's called.